market_next_door_api/market_next_door_api/urls.py

```diff
@@ -22,8 +22,12 @@
 
 urlpatterns = [
     path('customers/', views.customer_list),
-    path('customers/<int:id>', views.customer_details),
+    path('customers/<int:id>/', views.customer_details),
+    path('items/', views.item_list),
+    path('items/<int:id>/', views.item_details),
     path('vendors/', views.vendor_list),
-    path('vendors/<int:id>', views.vendor_details),
+    path('vendors/<int:id>/', views.vendor_details),
+    path('vendors/<int:vendor_id>/items/', views.vendor_item_list),
+    path('vendors/<int:vendor_id>/items/<int:item_id>/', views.vendor_item_details),
```

Comment:
Great work here in seperating all items from vendor/items. Looks great!

Reply:
Question here - would I have to do the same for vendors as you did for items since a vendor is associated with a market?